packages/clerk-js/src/utils/captcha/turnstile.ts

```diff
@@ -166,12 +165,18 @@ export const getTurnstileToken = async (opts: CaptchaOptions) => {
             closeModal?.();
             resolve([token, id]);
           },
-          'before-interactive-callback': async () => {
+          'before-interactive-callback': () => {
             if (modalWrapperQuerySelector) {
               const el = document.querySelector(modalWrapperQuerySelector) as HTMLElement;
               el?.style.setProperty('visibility', 'visible');
               el?.style.setProperty('pointer-events', 'all');
-              return;
+            } else {
+              const visibleWidget = document.getElementById(CAPTCHA_ELEMENT_ID);
+              if (visibleWidget) {
+                visibleWidget.style.maxHeight = 'unset';
+                visibleWidget.style.minHeight = '68px'; // this is the height of the Turnstile widget
+                visibleWidget.style.marginBottom = '1.5rem';
```

Comment:
This utility does not have access to our UI design system, so we need to hardcode these values.